index.html

```diff
@@ -42,15 +42,13 @@
   <body data-cite="credential-management-1">
     <section id="abstract">
       <p>
-        This document specifies an API to allow user agents to mediate access
-        to, and representing, a verifiably-issued digital credential (e.g. a
-        government issued digital driver's license, a passport issued by a
-        country, a student card or diploma issued by a university, an employee
-        card issued by a company, a passenger's boarding pass issued by an
-        airline, a profile issued by an online social network, membership
-        cards, vaccination records, etc) license). The API builds on
-        [[[credential-management-1]]] as a means to request a digital
-        credential from the user agent or underlying platform.
+        This document specifies an API to enable [=user agents=] to mediate
+        access to, and present, a digital credential such as a driver's license,
+        a student ID card, a boarding pass, or
+        <a href="#credential-type-examples">other types of credential or
+        license</a>. The API builds on [[[credential-management-1]]] as a means
```

Comment:
```suggestion
        <a href="#credential-type-examples">other types of credential</a>.
        The API builds on [[[credential-management-1]]] as a means
```
digital version of the licenses have been treated as one type of credentials, so probably no need to explicitly call out license here